src/renderer/stores/chat.js

```diff
@@ -66,8 +66,7 @@ chatStore.reducers.push(({ type, payload, id }, state) => {
       messages: {
         ...state.messages,
         ...payload.messagesIncoming
-      },
-      scrollToBottomIfClose: true
```

Comment:
this line is needed: in my tests it only scrolls down if this is true. Why did you remove it?